src/stratigraphy/benchmark/metrics.py

```diff
@@ -58,23 +58,32 @@ def get_metrics_list(self) -> list[Metrics]:
         return list(self.metrics.values())
 
 
-class DatasetMetricsCatalog:
+class OverallMetricsCatalog:
     """Keeps track of all different relevant metrics that are computed for a dataset."""
 
-    def __init__(self):
-        self.metrics: dict[str, DatasetMetrics] = {}
+    def __init__(self, languages: list[str]):
```

Comment:
`languages`  has type `set[str]` where we call this constructor in predictions.py.

Reply:
I just checked as well, and you are correct. Thank you!